### PR TITLE
(chore) ESLint: strictness++, use shared config

### PR DESCRIPTION
**Pull request by jokeyrhyme**

### CODE REVIEW

File: forms/models/elements/location.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ define(function (require) {
           this.validationError = errors;
           this.trigger('invalid', this);
           return Promise.reject(error);
-        });
+        }.bind(this));
```
Comment (simonmarklar) [inline forms/models/elements/location.js]: with the removal of the bind, i think 

```
this.validationError = errors;
this.trigger('invalid', this);
```

should swap out `this` for `self` ??

Comment (simonmarklar) [inline forms/models/elements/location.js]: agh actually read that wrong

image

**Comment by jokeyrhyme** [comment]
Yeah, we already had `self` there, so I just used it instead of `this`.

**Comment by simonmarklar** [comment]
lgtm :+1:

**Pull request merged**